runtime/src/lib.rs

```diff
@@ -863,6 +873,67 @@ impl pallet_verifiers::Config<pallet_proofofsql_verifier::ProofOfSql<Runtime>> f
     >;
 }
 
+parameter_types! {
+    pub const Coprocessor: Option<StateMachine> = Some(StateMachine::Kusama(4009));
```

Comment:
This is Hyperbridge Parachain settling on Kusama right ? When in mainnet I guess we should switch this to "Polkadot" ?

Reply:
correct, this is `Polkadot(3367)` in mainnet